src/pages/ReimbursementAccount/CompleteVerification/substeps/ConfirmAgreements.js

```diff
@@ -21,16 +37,34 @@ const validate = (values) => {
         errors.certifyTrueInformation = 'completeVerificationStep.certifyTrueAndAccurateError';
     }
 
-    if (!ValidationUtils.isRequiredFulfilled(values.isControllingOfficer)) {
-        errors.isControllingOfficer = 'completeVerificationStep.isControllingOfficerError';
+    if (!ValidationUtils.isRequiredFulfilled(values.isAuthorizedToUseBankAccount)) {
+        errors.isAuthorizedToUseBankAccount = 'completeVerificationStep.isAuthorizedToUseBankAccountError';
     }
 
     return errors;
 };
 
-function ConfirmAgreements() {
+const completeVerificationKeys = CONST.BANK_ACCOUNT.COMPLETE_VERIFICATION.INPUT_KEY;
```

Comment:
NAB: in the previous file you save the same CONST.BANK_ACCOUNT.COMPLETE_VERIFICATION.INPUT_KEY as COMPLETE_VERIFICATION_KEYS  (in uppercase) and here in camelCase, maybe let's stick to the one convention?